src/main/groovy/org/broadinstitute/orsp/ConsentGroupExtraProperties.groovy

```diff
@@ -0,0 +1,81 @@
+package org.broadinstitute.orsp
+
+class ConsentGroupExtraProperties {
+    // General Data
+    String startDate = ""
+    String endDate = ""
+    String onGoingProcess = ""
+    String source = ""
+    String collInst = ""
+    String collContact = ""
+    String consent = ""
+    String protocol = ""
+    String institutionalSources = ""
+    String describeConsentGroup = ""
+    String requireMta = ""
+    String projectReviewApproved = "false"
+
+    // Info Security
+    String pii = ""
+    String compliance = ""
+    String textCompliance = ""
+    String sensitive = ""
+    String textSensitive = ""
+    String accessible = ""
+    String textAccessible = ""
+
+    // Sharing data
+    String sharingPlan = ""
+    String databaseControlled = ""
+    String databaseOpen = ""
+
+    // InternationalCohorts
+    String individualDataSourced = ""
+    String isLinkMaintained = ""
+    String feeForServiceWork = ""
+    String areSamplesComingFromEEAA = ""
+    String isCollaboratorProvidingGoodService = ""
+    String isConsentUnambiguous = ""
+
+    transient ConsentGroupExtraProperties buildConsentGroupExtraProperties(Map<String, List<String>> extraPropsMap) {
```

Comment:
I don't think you need `transient` here - that's used to indicate transience on domain objects that require saving to the db.